deployer/scripts/upgrade.sh

```diff
@@ -278,20 +286,103 @@ function getArrayIndex() {
   fi
 }
 
+# This function will wait until the latestVersion changes
+function waitForChange() {
+
+  local currentVersion=$1
+  local dc=$2
```

Comment:
doesn't have to be a DC, could be any single object... maybe just change name of var to `object`?